collection-scripts/gather_windows_node_logs

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Instead of this; would it be better to just merge this into one bash script and select the things we want using labels?

Reply:
Most of the Linux log collection relies on the fact that the service logs being collected are from journald. This is not the case for Windows, and why I felt it made sense to seperate them

Reply:
We are missing container runtime, logs (docker), as well as kublet how do we get those added?

Reply:
We want to collect everything required to debug what is happening on the node (service logs wise); as it pertains to OpenShift. So I think we want to collect at the very least the container runtime and the kublet process logs (thinks that are installed outside of OpenShift that are pre-requisites for OpenShift to run or be connected to the cluster). .

Reply:
@sferich888 collecting kubelet logs is doable. But as @sebsoto mentioned, collecting the Docker logs will be difficult. Please open an issue on our board so that we can track this.

Reply:
I've added the kubelet log.

The issue with the Docker logs is that the logging location for that is not under our control. Our product asks that our users bring their own Windows VM image, with docker pre-installed. We'd either have to increase the customer requirements, or investigate reconfiguring Docker.